PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchView.swift

```diff
@@ -19,7 +19,12 @@ struct SearchView: View {
 
 // MARK: - Search Results Component
 struct ResultsView: View {
+    @ObservedObject
+    var viewModel: SearchViewModel
     var results: [SearchItem]
+
+    @State var bannerData: BannerModifier.BannerData = BannerModifier.BannerData(image: .looking, title: "Limited search results", detail: "You can only search titles and URLs while offline. Connect to the internet to use Premium's full-text search.")
```

Comment:
what do you think about encapsulating this logic in the banner since the results view shouldn't have to know this detail?

Reply:
Because we want to make the banner reusable, having a level of separation would be good imo. I also dont think there is any logic involved other than passing in the image, and two strings.

Reply:
sounds good! Do you think this would be better to add to the `viewModel` so that we don't have the data in the View. A `BannerViewModel` may be overkill for now. Don't have to do this now, but curious to hear your perspective.

Reply:
I dont think it would provide much benefit with having a VM, but I'm open to doing it if needed.  It just seems like too much separation imo.  happy to talk further about when/what we should use a VM for though.

Reply:
ok! lets keep this as is for now